plugins/convert_README_to_header.sh

```diff
@@ -30,8 +30,8 @@ outfilename=`basename $outfile`;
 # See http://stackoverflow.com/a/16576291
 # On Mac OS's sed, \n is not recognized as a newline character, but
 # \[actual newline] works
-desc=`sed -e ':a' -e 'N' -e '$!ba' -e 's/\"/\\\"/g' -e 's/\n/\\\\n"\\
-"/g' "$path/README.md"`;
+desc=`sed -e ':a' -e 'N' -e '$!ba' -e 's#\\\#\\\\\\\#g' -e 's#\"#\\\"#g' -e 's#\n#\\\\n"\\
```

Comment:
This feels like the wrong number of backslashes to me, or is it one for the shell and one for C++?

Reply:
This was reached by trial and error, regex are black magic :P
But what it does is it changes a single \ in the README to a \\ (escaped \) in the header file.
\ also needs to be escaped in the regex and I think also for the bash script running it which is what ends up being 3...

Reply:
:O even here you need to escape you \\ interesting....
changes \\ to \\\